### ISSUE TITLE
Inspircd S2S UID command error from specific ISP

### ISSUE THREAD

**Issue by ping86** [inspircd/inspircd#1033]
I am using inspircd 2.0.18 and when a user connect from this ISP (axtel.net) inspircd return the following error:

```
Error handling S2S command :326 UID 326BAXBFL 1429109071 gest688114 148-240-xxx-xx .ftth.dynamic.axtel.net chat-e91.nhu.8ffsj4.IP fl_cgs 148.240.xxx.xxx 1429109076 +x :realname
```

Protocol specification:
------------------------------
```
:<sid of new users server> UID <uid> <timestamp> <nick> <hostname> <displayed-hostname> <ident> <ip> <signon time> +<modes {mode params}> :<gecos>
```
User connection:
----------------------
```
:326 UID 326BAXBFL 1429109071 gest688114 148-240-xxx-xx .ftth.dynamic.axtel.net chat-e91.nhu.8ffsj4.IP fl_cgs 148.240.xxx.xxx 1429109076 +x :realname
```
Visual comparation
--------------------------
![inpircd-uid](https://cloud.githubusercontent.com/assets/1170773/7176282/41cff1c4-e41d-11e4-8987-c9ed95b38036.png)

You can see a white space between  "148-240-xxx-xx" ".ftth.dynamic.axtel.net"

Why the hostname has white space? Is it possible inspircd bug? When this error occurred the two servers had a netsplit.

Thanks by advance.

**Comment by ping86** [comment]
Also if I do host command for this ip, you can see a "\032" character (whitespace character).

```
host 148.240.xxx.xxx
xxx.xxx.240.148.in-addr.arpa domain name pointer 148-240-xxx-xxx\032.ftth.dynamic.axtel.net.
```

**Comment by Sheogorath-SI** [comment]
it's not an inspircd error:
http://148.240.67.96.ipaddress.com/

it's really a whitespace

**Comment by ping86** [comment]
We should catch this ISP error to prevent inspircd failures.

**Comment by Sheogorath-SI** [comment]
I would say it should use IP instead of hostname if there is a whitespace in it. Yes.

**Comment by md-5** [comment]
Yes I'm sure someone in the team will work on it soon.
We are discussing it in IRC now.

**Comment by ping86** [comment]
If someone have the same problem, set nouserdns="yes" until the error is corrected by the inspircd team

**Comment by SaberUK** [comment]
@ping86 Can you test #1035?

**Comment by attilamolnar** [comment]
Thanks for the report, fixed in 2.0.19.
